### ISSUE TITLE
Arcus usability in .NET Core 3 projects

### ISSUE THREAD

**Issue by fgheysels** [arcus-azure/arcus.webapi#123]
**Is your feature request related to a problem? Please describe.**
.NET Core 3 uses another concept then .NET Core 2.2 when it comes to package-references, especially to references to Microsoft.Extensions.* and Microsoft.AspNetCore packages

Are the Arcus libs usable in both .NET Core 2.* and .NET Core 3.* projects ?
Shouldnt the package references of the Arcus libs be changed to something as is explained in [this](https://andrewlock.net/converting-a-netstandard-2-library-to-netcore-3/#libraries-that-depend-on-asp-net-core-nuget-packages) article ?

**Comment by fgheysels** [comment]
Tried something with Arcus.WebApi.Logging; check this pr https://github.com/arcus-azure/arcus.webapi/pull/124

**Comment by tomkerkhove** [comment]
Everything should be usable with 2.x & 3.x but it's certainly not ideal, we should align those indeed.

**Comment by tomkerkhove** [comment]
@stijnmoreels Would you mind going through all our projects to see if we align wit the guidance on https://andrewlock.net/converting-a-netstandard-2-library-to-netcore-3/#libraries-that-depend-on-asp-net-core-nuget-packages please?

**Comment by fgheysels** [comment]
Should we use `netcoreapp3.0` in the TargetFrameworks section, or should be use `netstandard2.1`

https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-implementation-support

**Comment by tomkerkhove** [comment]
Agreed, would be good to do this asap @stijnmoreels - Can you pick this up or not?
